test/test_images/sendevents/main.go

```diff
@@ -106,13 +111,31 @@ func main() {
 		os.Exit(1)
 	}
 
-	t, err := cloudevents.NewHTTPTransport(
+	tOpts := []http.Option{
 		cloudevents.WithTarget(sink),
 		encodingOption,
-	)
+	}
+
+	// Add input tracing.
+	if addTracing {
+		tOpts = append(tOpts, http.WithMiddleware(tracing.HTTPSpanMiddleware))
```

Comment:
Does this Pod ever receive events? Or only send them? If only send, then we don't need this middleware added.

Reply:
You are right. After removing tracing headers are sent.